README.md

```diff
@@ -12,22 +12,6 @@
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
```

Comment:
- Good job attempting previously suggested changes by NY colleague. :muscle: 
- [ ] However, it appears there's a misunderstanding with what he suggested you delete. The reviewer suggested deleting the `Live Demo` (because you don't have a project with a live link yet) and then to remove the `Run tests` and `Deployment` because you do not have them in your ReadMe file in order to avoid confusing users by providing them with dead links. So, I think you have to add all others back to code - `Built with`, `Tech stack`, etc. since you have them in your file.

Reply:
Thank you, It is done. 
I have removed also the `FAQ` link because it does not have contents in the Readme file